solidity/integration/stress_test.js

```diff
@@ -0,0 +1,96 @@
+const BondedECDSAKeepVendor = artifacts.require("./BondedECDSAKeepVendor.sol")
+const BondedECDSAKeepVendorImplV1 = artifacts.require(
+  "./BondedECDSAKeepVendorImplV1.sol"
+)
+const BondedECDSAKeepFactory = artifacts.require("./BondedECDSAKeepFactory.sol")
+const BondedECDSAKeep = artifacts.require("./BondedECDSAKeep.sol")
+
+// how many iterations
+const iterations = 10
+// delay between the iterations [ms]
+const delay = 30000
+// number of keeps created at once
+const stressLevel = 5
+
+const groupSize = 3
+const threshold = 3
+const bond = 10
+
+module.exports = async function () {
+  const accounts = await web3.eth.getAccounts()
+
+  const keepOwner = accounts[4]
+  const application = accounts[5]
+
+  let keepFactory
+
+  try {
+    const keepVendor = await BondedECDSAKeepVendorImplV1.at(
+      (await BondedECDSAKeepVendor.deployed()).address
+    )
+    const keepFactoryAddress = await keepVendor.selectFactory()
+    keepFactory = await BondedECDSAKeepFactory.at(keepFactoryAddress)
```

Comment:
Maybe in this test we don't need to call the vendor?
```suggestion
    keepFactory = await BondedECDSAKeepFactory.deployed()
```

Reply:
Yep, good point. It can simplify things. Please see https://github.com/keep-network/keep-ecdsa/pull/357/commits/2751f092af5509cec187a83331ec2ce8b6749d26